src/resolvers/donationResolver.test.ts

```diff
@@ -170,6 +170,63 @@ function donationsByProjectIdTestCases() {
       DONATION_SEED_DATA.FIFTH_DONATION.id,
     );
   });
+  it('should search by donation amount', async () => {
+    const result = await axios.post(
+      graphqlUrl,
+      {
+        query: fetchDonationsByProjectIdQuery,
+        variables: {
+          projectId: SEED_DATA.FIRST_PROJECT.id,
+          searchTerm: '100',
+        },
+      },
+      {},
+    );
+
+    const donations = result.data.data.donationsByProjectId.donations;
+    assert.equal(
+      Number(donations[0]?.id),
```

Comment:
@CarlosQ96  I think it should be better instead of checking id of returned donation you can check the amount of returned donation, or other test case you can check equality of donation.currency, because when you are checking the id the test would fail if create donations with same amount or same currency in other part of our test codes, so it can't be reliable